docs/API.md

```diff
@@ -582,3 +757,21 @@ GET https://trade-service.wealthsimple.com/bank-accounts
     ]
 }
 ```
+
+### Get Foreign Exchange Rate
+#### Request
+```
+GET https://trade-service.wealthsimple.com/forex
+```
+
+#### Response
+```
+{
+    "USD": {
+        "buy_rate": 1.3651,
```

Comment:
Lol. Can you transfer in USD to these accounts or are just doomed to deal with this spread?

Reply:
You cannot transfer in USD. Pretty sure thats how they make money right now. Everything goes CAD --> USD when buying US options